### DISCUSSION TITLE
How to customize the logger in Trainer for detailed graphs

### DISCUSSION THREAD

**Discussion by cemox35** [asteroid-team/asteroid#560] [Q&A]
Hello there,

I want to loss/epoch, acc graphs, etc. However, I only see the tensorboard loss/step and val_loss/step. How can I customize/add it?

Thank you.

**Comment by mpariente** [comment]
By subclassing `System` and using the logging logic of lightning (using `self.log`).

**Reply by cemox35** [reply]
I used the predefined Conv-Tasnet model. Also, I am beginner. Can you show some code examples? Thank you for your help.

**Reply by mpariente** [reply]
Have a look at the docs of pytorch-lightning, and try to make a subclass of System for logging. If you show me your attempt, I'll correct it.
